packages/trader/src/Modules/Trading/Components/Form/TradeParams/amount.jsx

```diff
@@ -50,7 +48,7 @@ const Amount = ({
             error_messages={validation_errors.amount}
             fractional_digits={getDecimalPlaces(currency)}
             id='amount'
-            inline_prefix={is_single_currency ? currency : null}
+            inline_prefix={is_single_currency ? currency : 'USD'}
```

Comment:
why are we hardcoding? anyway to get the default currency from API? perhaps from landing company call